### PR TITLE
Add WidgetControl example on the ipyleaflet notebook

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: notebooks/10.04-ipyleaflet.ipynb [hunk 1]
```diff
@@ -276,15 +276,45 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "widgets-tutorial",
+   "display_name": "Python 3",
```
Comment (mwcraig) [inline notebooks/10.04-ipyleaflet.ipynb]:
```suggestion
   "display_name": "widgets-tutorial",
```

File: notebooks/10.04-ipyleaflet.ipynb [hunk 2]
```diff
@@ -276,15 +276,45 @@
...
    "language": "python",
-   "name": "widgets-tutorial"
+   "name": "python3"
```
Comment (mwcraig) [inline notebooks/10.04-ipyleaflet.ipynb]:
```suggestion
   "name": "widgets-tutorial"
```

File: notebooks/10.04-ipyleaflet.ipynb (hunk 1 above)
Comment (martinRenou) [reply notebooks/10.04-ipyleaflet.ipynb]: Woaw this new Github feature is super cool!

**Comment by martinRenou** [comment]
Thanks for the review :)

**Comment by mwcraig** [comment]
Thanks, merging!

**Pull request merged**